carbon-design-system/carbon
# fix: dropdown contrast focus styles

Closes #12720 

Paired up with @francinelucca - got a good solution going 🎉

<img width="906" alt="Screenshot 2023-06-15 at 17 10 07" src="https://github.com/carbon-design-system/carbon/assets/32720851/11602c8b-40a6-4caf-88f2-cb2edaea43a0">

#### Testing / Reviewing
- This must be tested with keyboard navigation 
- Make sure Dropdown, Combox & Multiselect have the above focus styling and the light-grey background has been removed - through all the different variants
- Hover styles should remain the same on mouse
- @kingtraceyj tagging you for a visual review :)
